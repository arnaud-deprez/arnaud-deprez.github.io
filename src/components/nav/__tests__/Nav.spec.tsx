import React from 'react'
import { render } from '@testing-library/react'
import { Nav as BootstrapNav } from 'react-bootstrap'
import { Nav } from '..'

describe('<Nav/>', () => {
  it('render a list of nav items', () => {
    const { asFragment } = render(
      <Nav>
        <BootstrapNav.Link>Menu 1</BootstrapNav.Link>
        <BootstrapNav.Link>Menu 2</BootstrapNav.Link>
        <BootstrapNav.Link>Menu 3</BootstrapNav.Link>
      </Nav>
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
