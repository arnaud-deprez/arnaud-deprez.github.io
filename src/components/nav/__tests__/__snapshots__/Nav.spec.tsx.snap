// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav/> render a list of nav items 1`] = `
<DocumentFragment>
  <ul
    class="nav"
  >
    <li
      class="nav-item"
    >
      <a
        class="nav-link"
        href="#"
        role="tab"
        tabindex="-1"
      >
        Menu 1
      </a>
    </li>
    <li
      class="nav-item"
    >
      <a
        class="nav-link"
        href="#"
        role="tab"
        tabindex="-1"
      >
        Menu 2
      </a>
    </li>
    <li
      class="nav-item"
    >
      <a
        class="nav-link"
        href="#"
        role="tab"
        tabindex="-1"
      >
        Menu 3
      </a>
    </li>
  </ul>
</DocumentFragment>
`;
